src/ImageView.jl

```diff
@@ -476,11 +476,11 @@ function histsignals(enabled::Signal, defaultimg, img::Signal, clim::Signal{CLim
     return histsigs
 end
 
-function scaleminmax(mtyp::DataType, cmin::AbstractRGB{T}, cmax::AbstractRGB{T}) where {T}
+function scaleminmax(::Type{Tout}, cmin::AbstractRGB{T}, cmax::AbstractRGB{T}) where {T,Tout}
     r = scaleminmax(T, red(cmin), red(cmax))
     g = scaleminmax(T, green(cmin), green(cmax))
     b = scaleminmax(T, blue(cmin), blue(cmax))
-    return x->mtyp(r(red(x)), g(green(x)), b(blue(x)))
+    return x->Tout(nanz(r(red(x))), nanz(g(green(x))), nanz(b(blue(x))))
```

Comment:
This fixes an error when displaying [this test image](https://github.com/JuliaImages/ImageView.jl/blob/a940b609ae2f066015b3d0486959754731d92980/test/simple.jl#L23-L28).